### DISCUSSION TITLE
Any clues what happened during this flight?

### DISCUSSION THREAD

**Discussion by eben80** [dl9rdz/rdz_ttgo_sonde#382] [Q&A] [answered]
https://sondehub.org/?sondehub=1#!mt=Mapnik&mz=12&qm=3h&mc=49.07522,21.21958&f=V2110990&q=V2110990
Did the sonde run out  of gps accuracy over a certain altitude?

**Comment by hr-ru** [comment, marked as answer]
Maybe the GPS chip was faulty, possibly set in the wrong dynamic platform model.
If not in flight mode, it will more or less behave as observed.

**Reply by eben80** [reply]
I guess I don't know what is actually set on the sondes before release. I know about the calibration but I guess there's much more going on. That NFC communication is pretty interesting too. I need to try and look at it with my Flipper Zero.

**Comment by pavolgaj** [comment]
Maybe, something similar to these flights https://github.com/dl9rdz/rdz_ttgo_sonde/discussions/204
